.gitignore

```diff
@@ -2,3 +2,5 @@
 **/*.rs.bk
 Cargo.lock
 *~
+/*.txt
+/*.zip
```

Comment:
this can be added in your global .gitignore

Reply:
we don't have any txt or zip for the moment, I don't see what is the problem

Reply:
1) only you have .txt and .zip in navitia_model directory.
All your "personal" files can be added in ~/.gitignore

2) later we will have .txt files for tests. We will need to rollback this change.

3) I also have ".out" and ".orig" and more files but I don't add them in this .gitignore 

but anyway it is not critical

Reply:
ok, I've reverted the modification